documentation/install.rst

```diff
@@ -1,5 +1,18 @@
-Installation
-============
+Installation with pre-built wheels
+==================================
+
+The easiest way to install Binder is to use pre-built wheels from PyPI. To install Binder, run:
+
+.. code-block:: bash
+
+    pip install cppbinder
```

Comment:
Have you checked how well this works when deployment system is different (ie lib stdc++ is different) from the the one you build wheels one? For my projects this was significant issue so i end up just rebuilding Binder on target system.